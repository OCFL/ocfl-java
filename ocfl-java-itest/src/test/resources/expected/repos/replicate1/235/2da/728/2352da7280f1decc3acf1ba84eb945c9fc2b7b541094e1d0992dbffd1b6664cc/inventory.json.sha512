30f0812a0b318aff898fb9d7dc5f48f65798cfdc468981a0c7c8192ea872b342d43d42b5272bdbc94a45a484b7927d010a148541c8fdbe0a1434d4e40eef3d48  inventory.json
