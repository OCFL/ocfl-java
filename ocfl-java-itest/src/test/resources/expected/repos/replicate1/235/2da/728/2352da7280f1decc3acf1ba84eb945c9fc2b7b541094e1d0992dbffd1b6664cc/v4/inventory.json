{
  "id" : "o1",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v4",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "3bafbf08882a2d10133093a1b8433f50563b93c14acd05b79028eb1d12799027241450980651994501423a66c276ae26c43b739bc65c4e16b10c3af6c202aebb" : [ "v2/content/f3" ],
    "40b244112641dd78dd4f93b6c9190dd46e0099194d5a44257b7efad6ef9ff4683da1eda0244448cb343aa688f5d3efd7314dafe580ac0bcbf115aeca9e8dc114" : [ "v1/content/f2" ],
    "4dff4ea340f0a823f15d3f4f01ab62eae0e5da579ccb851f8db9dfe84c58b2b37b89903a740e1ee172da793a6e79d560e5f7f9bd058a12a280433ed6fa46510a" : [ "v1/content/f1" ],
    "7db70149dac5561e411a202629d06832b06b7e8dfef61086ff9e0922459fbe14a69d565cf838fd43681fdb29a698bfe377861b966d12416298997843820bfdb7" : [ "v3/content/f2" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "1",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "40b244112641dd78dd4f93b6c9190dd46e0099194d5a44257b7efad6ef9ff4683da1eda0244448cb343aa688f5d3efd7314dafe580ac0bcbf115aeca9e8dc114" : [ "f2" ],
        "4dff4ea340f0a823f15d3f4f01ab62eae0e5da579ccb851f8db9dfe84c58b2b37b89903a740e1ee172da793a6e79d560e5f7f9bd058a12a280433ed6fa46510a" : [ "f1" ]
      }
    },
    "v2" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "2",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "3bafbf08882a2d10133093a1b8433f50563b93c14acd05b79028eb1d12799027241450980651994501423a66c276ae26c43b739bc65c4e16b10c3af6c202aebb" : [ "f3" ],
        "40b244112641dd78dd4f93b6c9190dd46e0099194d5a44257b7efad6ef9ff4683da1eda0244448cb343aa688f5d3efd7314dafe580ac0bcbf115aeca9e8dc114" : [ "f2" ]
      }
    },
    "v3" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "3",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "3bafbf08882a2d10133093a1b8433f50563b93c14acd05b79028eb1d12799027241450980651994501423a66c276ae26c43b739bc65c4e16b10c3af6c202aebb" : [ "f3" ],
        "4dff4ea340f0a823f15d3f4f01ab62eae0e5da579ccb851f8db9dfe84c58b2b37b89903a740e1ee172da793a6e79d560e5f7f9bd058a12a280433ed6fa46510a" : [ "f1" ],
        "7db70149dac5561e411a202629d06832b06b7e8dfef61086ff9e0922459fbe14a69d565cf838fd43681fdb29a698bfe377861b966d12416298997843820bfdb7" : [ "f2" ]
      }
    },
    "v4" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "replicate",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "40b244112641dd78dd4f93b6c9190dd46e0099194d5a44257b7efad6ef9ff4683da1eda0244448cb343aa688f5d3efd7314dafe580ac0bcbf115aeca9e8dc114" : [ "f2" ],
        "4dff4ea340f0a823f15d3f4f01ab62eae0e5da579ccb851f8db9dfe84c58b2b37b89903a740e1ee172da793a6e79d560e5f7f9bd058a12a280433ed6fa46510a" : [ "f1" ]
      }
    }
  }
}